WordPress/src/main/res/layout/backup_download_list_checkbox_item.xml

```diff
@@ -0,0 +1,38 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
1) Question (❓): This is identical to `backup_details_list_item.xml` file, isn't it? Maybe this duplication should be dealt with now. Wdyt?
2) Minor (🔍 ): Consider reformatting this layout file, I believe they are not in the right format. PS: I think this apply to all the new layouts.

Reply:
> This is identical to backup_details_list_item.xml file, isn't it? Maybe this duplication should be dealt with now. Wdyt?

This file no longer exists in a future branch. I am 3 branches ahead of this one locally.